tests/test_flows.py

```diff
@@ -401,18 +401,92 @@ def test_channel_selection(self):
         self.check_log(self.repeater1, "agent_wlan0", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
         self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
 
-        self.debug("Send channel selection request")
-        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x8006)
-        time.sleep(1)
-        self.debug("Confirming channel selection request has been received on agent")
-        self.check_log(self.repeater1, "agent_wlan0", "CHANNEL_SELECTION_REQUEST_MESSAGE")
-        self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_SELECTION_REQUEST_MESSAGE")
-
-        #self.debug("Confirming 1905.1 Ack Message request was received on agent")
-        # TODO: When creating handler for the ACK message on the agent, replace lookup of this string
-        # TODO: currently controller sends empty channel selection request, so no switch is performed
-        # self.check_log(self.repeater1, "agent_wlan0", "ACK_MESSAGE")
-        # self.check_log(self.repeater1, "agent_wlan2", "ACK_MESSAGE")
+        payload_empty="0x14"
+
+        # payload_wlan0 - request for change channel on 6
+        payload_wlan0=' '.join([
```

Comment:
There is nothing to gain by splitting this to multiple lines - can you keep it in 2-3 lines?
Or even better - add it to another yaml file which you can import from this file which can parse into a dictionary of messages:
```yaml
tlv_data:
    channel_selection:
         channel6: [...],
         channel36: [...],
         ...
    autoconfig:
        ...
};
```

Reply:
I have chosen 1 option - 3 strings with payload.